### DISCUSSION TITLE
How can I remove the noise from image

### DISCUSSION THREAD

**Discussion by tonyfkchu** [dlemstra/Magick.NET#673] [Help]
Dear Expert

Any way for me to remove the noise or the stamp of this this image
![image](https://user-images.githubusercontent.com/6896544/85188432-9ae71a80-b2d0-11ea-81cb-da38df161c49.png)

and the Original image is 
![audited financial statement with qualified audit opinion_Page_09](https://user-images.githubusercontent.com/6896544/85188453-cff36d00-b2d0-11ea-89d0-274b282ccbe2.png)

Many Thanks
Best Regards
Tony Chu

**Comment by dlemstra** [comment]
Could you help use out with how we can do this on the command line @fmw42?

**Comment by fmw42** [comment]
Are you trying to remove the stamp?  You said noise or stamp and that puzzles me. I see no noise. So are you trying to remove the stamp or make the stamp more clear to read.  

If you just want to remove the stamp from the smaller cropped region about the best I can do without filling in the hole in the "A" letters and such would be:

```
convert text.png -alpha off -threshold 5% -type bilevel \
-define connected-components:exclude-header=true \
-define connected-components:area-threshold=5 \
-define connected-components:mean-color=true \
-connected-components 8 \
x.png

```

![x](https://user-images.githubusercontent.com/13458218/85195180-4b2f3180-b285-11ea-83a2-194aa88443de.png)
